conda-recipe/meta.yaml

```diff
@@ -58,29 +61,30 @@ requirements:
     - cython
     - jinja2
     - pybind11
-    - numpy {{ numpy }}
-    - impi-devel  # [not win]
+    - numpy
+    - mpich  # [not win]
```

Comment:
Why mpich instead of impi-devel?

Reply:
It was for testing purposes, mpi version doesn't make difference in public build of conda-recipe.